homework2/fastq_filtrator.py

```diff
@@ -0,0 +1,75 @@
+"""GC-check function"""
+def GC_count_fun(f, gc_bounds = (0, 100)):
+  cnt=0
+  #Counting the amount of cytosine and guanine in a string
+  for nucl in f:
+    if (nucl == 'C') or (nucl == 'G'):
+      cnt += 1
```

Comment:
👍 не самый изящный, но зато самый эффективный способ посчитать GC. Кст, еще можно это немного упростить
```suggestion
  cnt=0
  #Counting the amount of cytosine and guanine in a string
  for nucl in f:
    cnt += (nucl == 'C' or nucl == 'G')
```
`f` - плохое имя